app/page.tsx

```diff
@@ -1,50 +1,190 @@
+import SeroMap from "../public/SeroMap.png";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faMosquito,
+  faVirus,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
```

Comment:
I also think font awesome has a much more robust collection of icons, so maybe we should switch to that one completely instead of lucide